travis/module.yml

```diff
@@ -58,8 +64,8 @@ before_script:
   - composer create-project altis/skeleton:dev-$TRAVIS_BRANCH --stability=dev --ignore-platform-req=php+ --ignore-platform-req=ext-* $HOME/test-root || composer create-project altis/skeleton:dev-master --stability=dev --ignore-platform-req=php+ --ignore-platform-req=ext-* $HOME/test-root || exit 1
   # Install Altis test theme package
   - cd $HOME/test-root && composer require altis/test-theme --ignore-platform-req=php+ --ignore-platform-req=ext-*
-  # Mark the test theme as the default
-  - cd $HOME/test-root && cat <<< $(jq '. + {"extra":{"altis":{"modules":{"cms":{"default-theme":"test-theme"}}}}}' composer.json | jq . -) > composer.json
+  # Mark the test theme as the default, and set PHP value from build matrix
+  - cd $HOME/test-root && jq --arg TRAVIS_PHP_VERSION $TRAVIS_PHP_VERSION '.config.platform.php = $TRAVIS_PHP_VERSION | .config.platform."ext-mbstring" = $TRAVIS_PHP_VERSION | .extra = {"altis":{"modules":{"cms":{"default-theme":"test-theme"},"local-server":{"php":$TRAVIS_PHP_VERSION}}}}' composer.json | cat > updated && mv updated composer.json
```

Comment:
Would it work to break this into 2 steps, it might simplify things and make it easier to identify where an issue might be if something goes wrong.

Reply:
Yup, that's the plan, was just testing as there is a weird issue with piping the final output to the file. Will do as things seem to be working now.